fix(ins): update routed ip docs (#2536)
scaleway/docs-content: compute/instances/how-to/migrate-routed-ips.mdx

  - compute
---

> During deployment, each Instance gets a NAT IP from its hosting node, besides its public IP, which is routed to the Instance through a [carrier-grade NAT](/compute/instances/concepts/#carrier-grade-nat-(cgnat)) solution. However, if the underlying hypervisor changes, the NAT IP changes. This can happen when you restart your Instance. The NAT IP lets Instances communicate via the underlying private network, without public IPs. 

Reviewer: ```suggestion
During deployment, each Instance gets a NAT IP from its hosting node, besides its public IP, which is routed to the Instance through a [carrier-grade NAT](/compute/instances/concepts/#carrier-grade-nat-(cgnat)) solution. However, if the underlying hypervisor changes, the NAT IP changes. This can happen when you restart your Instance. The NAT IP lets Instances communicate via an underlying internal network, without public IPs. 
```

  - You have [created an Instance](/compute/instances/how-to/create-an-instance/) that is not moved to routed IPs
</Message>

## Updating the required packages

To update the `scaleway-ecosystem` and `cloud-init` packages, [log into your Instance using SSH](/compute/instances/how-to/connect-to-instance/) and run the following command.

> * For Ubuntu and Dedbian Instances, you can use the following command:

Reviewer 2: ```suggestion
* For Ubuntu and Debian Instances, you can use the following command:
```

    ```bash

>     apt update scaleway-ecosystem cloud-init -y

Reviewer: ```suggestion
    apt upgrade scaleway-ecosystem cloud-init -y
```

    ```

> * For CentOS and Fedora systems, you can use the following command:

Author: ```suggestion
* For CentOS and Fedora Instances, you can use the following command:
```

    ```bash

>     apt update scaleway-ecosystem cloud-init -y

Reviewer: ```suggestion
    dnf update scaleway-ecosystem cloud-init -y
```
